front-end/src/tests/E2E-tests/04_commentTests.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect, Page } from '@playwright/test';
+import { PostPage } from '../pages/postPage';
+import { tooLargePostContent, validPostContent } from '../mock-data/post';
+import { deleteCommentRepositoryByContent } from '../../../../back-end/src/APIs/repositories/commentRepository';
+
+const setupPostPage = async (page: Page) => {
+	const postPage = new PostPage(page);
+	await postPage.gotoBase();
+	await postPage.check();
+	return postPage;
+};
+
+test.only('Create comment functionality works correctly with valid data', async ({ page }) => {
+	const postPage = await setupPostPage(page);
+
+	await postPage.gotoPost('1');
```

Comment:
Might fail due to post id not present in database. In database populating, the id of posts are not set, they're set automatically and if increment is not reset, the first post wouldn't be with id = 1. I think you might need to navigate to the post by ui.